src/sync_template.sh

```diff
@@ -65,10 +70,12 @@ echo "::endgroup::"
 echo "::group::commit and push changes"
 git add .
 
+if [[ -f ".github/$TEMPLATE_SYNC_IGNORE_FILE_NAME" ]]; then
+  TEMPLATE_SYNC_IGNORE_FILE_NAME=".github/$TEMPLATE_SYNC_IGNORE_FILE_NAME"
```

Comment:
just a small naming thing: It is not just a `file_name` anymore because it has also the `.github/` folder.
Maybe a possibility would to do smth like 

```bash
if [....]
   TEMPLATE_SYNC_IGNORE_FILE_PATH_PREFIX = ".github/"
...
fi
```

and when you do the actions, you add that variable in front of the file name

```bash
  "${TEMPLATE_SYNC_IGNORE_PATH_PREFIX}/${TEMPLATE_SYNC_IGNORE_FILE_NAME}
```

another solution would be to rename the variable to smth like

`TEMPLATE_SYNC_IGNORE_FILE_RELATIVE_PATH`

Reply:
I like renaming the variables. I renamed them to `*_FILE_PATH`. The logic was kept the same.